src/Microsoft.Health.Fhir.Api/Features/Filters/ValidateExportRequestFilterAttribute.cs

```diff
@@ -45,6 +45,7 @@ public ValidateExportRequestFilterAttribute()
                 KnownQueryParameterNames.Type,
                 KnownQueryParameterNames.Container,
                 KnownQueryParameterNames.Format,
+                KnownQueryParameterNames.TypeFilter,
```

Comment:
Do we want to validate that when `TypeFilter` is present, `Type` should also be part of the query parameters?

Reply:
The spec doesn't state that `Type` is required if `TypeFilter` is present. In theory you could have `TypeFilter` without `Type`.